integrations/mailer/mailer.py

```diff
@@ -471,7 +468,7 @@ def main():
             int: config.getint,
             float: config.getfloat,
             bool: config.getboolean,
-            list: lambda s, o: [e.strip() for e in config.get(s, o).split(',')]
+            list: lambda s, o: [e.strip() for e in config.get(s, o).split(',')] if len(config.get(s, o)) > 0 else []
```

Comment:
I suspect you don't need `> 0` as `0` will be false and `1` or more will be true.

Reply:
You're right (I am just used to do explicit rather than implicit).
Changed in last commit.